src/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Page transitions replaced by ZoomPageTransitionsBuilder
+description: Using the latest page transition instead of the old one.
+---
+
+## Summary
+
+In order to keep libraries follow the latest OEM behavior,
+we've replaced the default page transition builders from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` on all platforms excluding iOS and macOS.
+
+## Context
+
+`FadeUpwardsPageTransitionsBuilder` came with the first
+Flutter release, which defines a page transition that's
+similar to the one provided by Android O.
+
+`ZoomPageTransitionsBuilder` defines a page transition that's
+similar to the one provided by Android Q and R.
+
+According to the [Style guide for Flutter repo][], we've switched
+the default page transitions which mainly for all platforms excluded iOS and macOS.
+When the `PageTransitionsBuilder` is not found on the current platform,
+`ZoomPageTransitionsBuilder` will be the default.
+
+[Style guide for Flutter repo]: {{site.repo.flutter}}/wiki/Style-guide-for-Flutter-repo
+
+## Description of change
+
+`PageTransitionsBuilder`s defined in
+`PageTransitionsTheme._defaultBuilders` have changed from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` for `TargetPlatform.android`,
+`TargetPlatform.linux` and `TargetPlatform.windows`.
+
+## Migration guide
+
+If you want to switch back to the previous page transition builder
+(`FadeUpwardsPageTransitionsBuilder`), you should define builders
+explicitly for the target platforms.
+
+Code before migration:
+
+<!-- skip -->
+```dart
+MaterialApp(
+  theme: ThemeData(primarySwatch: Colors.blue),
+)
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+MaterialApp(
+  theme: ThemeData(
+    primarySwatch: Colors.blue,
+    pageTransitionsTheme: const PageTransitionsTheme(
+      builders: <TargetPlatform, PageTransitionsBuilder>{
+        TargetPlatform.android: FadeUpwardsPageTransitionsBuilder(), // Apply this to every platforms you need.
```

Comment:
is there a way to apply it to all platforms at once rather than having to list them all?

Reply:
Should we indicate that here?